### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by geotheory**
Basic working script to monitor Buendia OpenMRS server status.  Monitors status in 2 ways:
1. externally pings Buendia's server URL's to determine if they're responding
2. reads file `/home/buendia/server_status.txt`, containing single digit number code reporting OpenMRS's internal status report (to be implemented)

### CODE REVIEW

File: server_status.py [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+time.sleep(1)
+red_on()
+
+buendia = 0     # localhost status
+openmrs = 0     # openmrs server status
+openmrs_i = -1  # openmrs internal status
+last_check = time.time()
+
+# reset openmrs internal status
+subprocess.call('echo -1 > /home/root/debian/home/buendia/server_status.txt', shell=True)
```
Comment (zestyping) [inline server_status.py]: Is there a reason not to use the existing status reporting mechanism here? See the implementation of the http://server/status page in the buendia-dashboard package.

File: server_status.py [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    return 1
+  else: # other response (e.g. redirected) [Might have to debug this..]
+    return 2
+
+
+# INITIALISE SETTINGS
+
+red = False
+red_led = mraa.Gpio(31)     # 'GP44'
+red_led.dir(mraa.DIR_OUT)
```
Comment (zestyping) [inline server_status.py]: If we really need to record the state of each LED, then encapsulate it in a class.

I don't see why we need to record the on/off state though.  I assume sending an "on" command to an LED that's already on is harmless and takes negligible time.  For simplicity can we reduce the entire API to:

```
set_led(identifier, state)

identifier: RED, GREEN, or BLUE
state: True or False
```

File: server_status.py [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+os.chdir('/home/root/gpio')
+print "*** START server_status.py ***"
+
+# FUNCTIONS
+
+# Returns current line number
+def lineno(): 
+    return inspect.currentframe().f_back.f_lineno
+
+def red_on():
```
Comment (zestyping) [inline server_status.py]: Lots of code duplication here.  Let's write one function instead of six.

**Comment by dancunningham** [comment]
Thanks @zestyping for reviewing this! The new PR #2 is much cleaner and easier to understand!

**Pull request closed without merging**